### DISCUSSION TITLE
Do we have plan to support eclipse plugin

### DISCUSSION THREAD

**Discussion by AlwaysNoobCoder** [metalbear-co/mirrord#2223] [Polls]
As a deep Eclipse user, I am reluctant to migrate to IntelliJ. I tried
IntelliJ, but the cost of migration is too high, and many things are not known how to be done in IntelliJ. Moreover, in comparison, I feel that the current Eclipse takes up less memory and responds faster.

**Comment by aviramha** [comment]
Would be happy to see what people think. In any case, mirrord can probably used with Eclipse with a bit of configuration - create a run configuration that runs `mirrord exec -f CONFIG-FILE -- YOUR_EXEC YOUR ARGS` and it should work.
The only thing you'd be missing is the target selection dialog and improved UX.
